sdk/python/docs/TFJobClient.md

```diff
@@ -0,0 +1,171 @@
+
+### Example
+
+```python
+from kubernetes.client import V1PodTemplateSpec
+from kubernetes.client import V1ObjectMeta
+from kubernetes.client import V1PodSpec
+from kubernetes.client import V1Container
+
+from kubeflow.tfjob import constants
+from kubeflow.tfjob import utils
+from kubeflow.tfjob import V1ReplicaSpec
+from kubeflow.tfjob import V1TFJob
+from kubeflow.tfjob import V1TFJobList
+from kubeflow.tfjob import V1TFJobSpec
+from kubeflow.tfjob import TFJobClient
+
+
+container = V1Container(
+    name="tensorflow",
+    image="gcr.io/kubeflow-ci/tf-mnist-with-summaries:1.0",
+    command=[
+        "python",
+        "/var/tf_mnist/mnist_with_summaries.py",
+        "--log_dir=/train/logs", "--learning_rate=0.01",
+        "--batch_size=150"
+        ]
+)
+
+worker = V1ReplicaSpec(
+    replicas=1,
+    restart_policy="Never",
+    template=V1PodTemplateSpec(
+        spec=V1PodSpec(
+            containers=[container]
+        )
+    )
+)
+
+tfjob = V1TFJob(
+    api_version="kubeflow.org/v1",
+    kind="TFJob",
+    metadata=V1ObjectMeta(name="mnist",namespace=namespace),
+    spec=V1TFJobSpec(
+        clean_pod_policy="None",
+        tf_replica_specs={"Worker": worker}
+    )
+)
+
+
+tfjob_client = TFJobClient()
+tfjob_client.create(tfjob)
+
+```
+
+
+### Parameters
+Name | Type |  Description | Notes
+------------ | ------------- | ------------- | -------------
+tfjob  | [V1TFJob](V1TFJob.md) | tfjob defination| Required |
+namespace | str | Namespace for tfjob deploying to. If the `namespace` is not defined, will align with tfjob definition, or use current or default namespace if namespace is not specified in tfjob definition.  | Optional |
+
+### Return type
+object
+
+## get
+> get(name=None, namespace=None)
+
+Get the created tfjob in the specified namespace
+
+### Example
+
+```python
+from kubeflow.tfjob import TFJobClient
+
+tfjob_client = TFJobClient()
+tfjob_client.get('mnist', namespace='kubeflow')
+```
+
+### Parameters
+Name | Type |  Description | Notes
+------------ | ------------- | ------------- | -------------
+name  | str | tfjob name. If the `name` is not specified, it will get all tfjobs in the namespace.| Optional. |
+namespace | str | The tfjob's namespace. Defaults to current or default namespace.| Optional |
+
+
+### Return type
+object
+
+
+## patch
+> patch(name, tfjob, namespace=None)
+
+Patch the created tfjob in the specified namespace.
+
+Note that if you want to set the field from existing value to `None`, `patch` API may not work, you need to use [replace](#replace) API to remove the field value.
+
+### Example
+
+```python
+
+tfjob = V1TFJob(
+    api_version="kubeflow.org/v1",
+    ... #update something in TFJob spec
+)
+
+tfjob_client = TFJobClient()
+tfjob_client.patch('mnist', isvc)
+
+```
+
+### Parameters
+Name | Type |  Description | Notes
+------------ | ------------- | ------------- | -------------
+tfjob  | [V1TFob](V1TFJob.md) | tfjob defination| Required |
```

Comment:
nit: V1TFob to V1TFJob

Reply:
Did you write the doc or generate it automatically? @jinchihe

Reply:
Some of them is generated automatically while some of are drafted muanully. Please see the PR description section that I show what's done in there. You know there is no way to generate Client Apis from openapi and swagger file :-) @gaocegege

Reply:
Updated. Thanks @nrchakradhar